### DISCUSSION TITLE
腾讯云产品搜索框无法输入，你们有遇到吗？

### DISCUSSION THREAD

**Discussion by Ansen** [rime/home#753] [Q&A]
> 腾讯改版了，没这个问题了

~~腾讯云官网的产品搜索框无法输入汉字~~

版本：0.14.3
输入方案：五笔86

地址：https://cloud.tencent.com/

![图片](https://user-images.githubusercontent.com/676412/111936941-b0100480-8b01-11eb-9c27-62a523f08bcd.png)

**Comment by amorphobia** [comment]
这应该向腾讯反映吧

**Reply by Ansen** [reply]
但是使用系统自带的输入法是正常的

**Reply by TTTPOB** [reply]
我发现很多无法输入的输入框只需要先输入一个数字之后就可以用rime正常输入

**Comment by MokOopsing** [comment]
![image](https://user-images.githubusercontent.com/39441028/142430477-0132693c-1c9c-41c1-b975-a300128e1b63.png)

有没有换个流览器试试

**Reply by Ansen** [reply]
> 
> 
> ![image](https://user-images.githubusercontent.com/39441028/142430477-0132693c-1c9c-41c1-b975-a300128e1b63.png)
> 
> 有没有换个流览器试试

貌似腾讯改版了，没有这个bug了

**Comment by TTTPOB** [comment]
有遇到别的网站这样，[quartzy](app.quartzy.com) 的 add request框 (这个网站不注册是没办法进入到可以触发bug的页面的，不用试了)
还有 [telegram web k](web.telegram.org/k) 也是使用小狼毫没办法打字

**Comment by rtransformation** [comment]
没有问题。
![屏幕截图 2022-03-10 101117](https://user-images.githubusercontent.com/11702676/157574738-43405ff9-816e-4da4-aecb-ed731192cfde.png)
